nagios/config/wavefront-csp-token.cfg

```diff
@@ -0,0 +1,41 @@
+# Nagios command definitions for host and service metrics to the Wavefront server using CSP API Token
+define command {
+    command_name  wavefront-perf-host
+    command_line /usr/local/nagios/libexec/nagios-metrics-wf.py /usr/local/nagios/var/host-perfdata \
+                                                        --wf_server [WAVEFRONT_URL] \
+                                                        --csp_api_token [CSP_API_TOKEN] \
+                                                        --csp_base_url [CSP_BASE_URL]
+}
+
+
+define command {
+    command_name  wavefront-perf-service
+    command_line  /usr/local/nagios/libexec/nagios-metrics-wf.py /usr/local/nagios/var/service-perfdata \
+                                                        --service \
+                                                        --wf_server [WAVEFRONT_URL] \
+                                                        --csp_api_token [CSP_API_TOKEN] \
+                                                        --csp_base_url [CSP_BASE_URL]
+}
+
+# Wavefront contact definition for event notification commands using CSP API Token
```

Comment:
Operations for Applications contact definition for event notification commands using a VMware Cloud services API token